plugin/pkg/scheduler/scheduler.go

```diff
@@ -184,6 +184,9 @@ func (sched *Scheduler) schedule(pod *v1.Pod) (string, error) {
 	return host, err
 }
 
+// preempt implements the preempt rules for pending pod and returns the suggested node.
```

Comment:
Please use this comment instead:
`preempt tries to create room for a pod that has failed to schedule, by preempting lower priority pods if possible.
If it succeeds, it adds the name of the node where preemption has happened to the pod annotations.
It returns the node name and an error if any.`